css/src/components/paper.css

```diff
@@ -61,4 +61,5 @@
 
 .yoast-paper__item:not(:first-of-type) {
```

Comment:
Just an idea (as this works fine)

To write out that every every item that is not the first item needs an margin on the top, we could also work with `.yoast-paper__item + .yoast-paper__item`.


Personally i think this is nicer. I do not think there is a good or bad way in this case. 

performance wise, i have no clue.